src/Components/test/testassets/BasicTestApp/ServerReliability/ReliabilityComponent.razor

```diff
@@ -0,0 +1,35 @@
+@inject Microsoft.JSInterop.IJSRuntime JSRuntime
+@using System.Security.Cryptography;
+@namespace BasicTestApp
+<h1>Counter</h1>
```

Comment:
Recently I've started putting a `<p>` at the top of each new E2E test case with a description of what scenario it's meant to test and how you could interact with it manually to explore the functionality. I think the E2E tests are doubly valuable when they aren't only intended for consumption by UI automation, but are also usable by devs investigating whether they can repro a certain bug.

I know in this case it's hard for humans to do anything useful with this component through the browser UI, since the whole point of it is to act as a target for misbehaving client apps. But there could be a description of this fact at the top, otherwise it seems bizarre to duplicate the "counter" functionality for the Nth time here.

Reply:
Done